supervision/README.md

```diff
@@ -42,6 +44,22 @@ During different periods, some activities will be more pronounced than others (e
        - There may also be TODOs for your supervisor. Prefix them with their initials, e.g., "[KB]".
   * If there are items that need preparation from the supervisor's end before the meeting, i.e., the questions/blockers block is not empty, make sure you share the agenda at least 24 hours before the meeting (by emailing the link to your Google document).
 
+### Supervisor roles and responsibilities 
+* Each thesis group has a course coordinator (fagligansvarlig) and a supervisor. 
+* The supervisor can be internal if the thesis work is carried out at the UiS or external if the thesis work is carried out at an external company.
+* Course coordinator and internal supervisor can be different people in case PhD students are the internal supervisors.
+* Course coordinator:
+  - Approve the level and scope of the project
+  - Sign a contract with the students before formal registration
+  - Approve the progress plan for the work
+  - Students are entitled to at least five meetings with the course coordinator throughout the semester.
+* Internal supervisors:
+  - Frequently meetings during the semester. Minimum one meeting every second week.
```

Comment:
Meets regularly with the student(s) during the semester (min. one meeting every two weeks).

Reply:
done